<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyGraf</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_row_btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Row</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListAdd"/>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_row_btn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove row</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListRemove"/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>261</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>5</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>5</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Column name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="show_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::VideoDisplay"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="color_edit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Table control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_file_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save file</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_file_tight_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>240</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save tight</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>&amp;Save as</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Q&amp;t</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
